source/classes/EnemyTemplate.js

```diff
@@ -44,6 +44,12 @@ class EnemyTemplate {
 	/** @type {import("discord.js").EmbedField} */
 	flavorText;
 
+	/** @param {number} integer */
+	setMaxHP(integer) {
```

Comment:
Using this to set the HP of spawned adds is pretty cool. Please mention it in the function documentation so it's easier to remember this functionality can be used that way.